EcpBtBot/DialogFlow/Response.py

```diff
@@ -19,7 +19,7 @@ def status(self):
 class Result:
     def __init__(self,result):
         self.result= result
-        self.action=result['action']
+        #self.action=result['action']
         self.source=result['source']
```

Comment:
what if you don't have 'source' in the input dictionary ? I would use result.get('source', None), so as to make sure it does not throw an exception.